packer/scylla_install_image

```diff
@@ -111,18 +111,17 @@ if __name__ == '__main__':
         run(f'curl -L -o /tmp/amazon-ssm-agent.deb https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/debian_{deb_arch()}/amazon-ssm-agent.deb', shell=True, check=True)
         run('dpkg -i /tmp/amazon-ssm-agent.deb', shell=True, check=True)
         run('systemctl enable amazon-ssm-agent', shell=True, check=True)
+        # Override chrony.conf to default setting
+        run('cp -a /etc/chrony/conf.d/00-cpc.conf /etc/chrony/chrony.conf', shell=True, check=True)
```

Comment:
where did you got the information that `/etc/chrony/conf.d/00-cpc.conf` is the correct configuration for each cloud ?

Reply:
yes it is